packages/cli/src/metro-config.ts

```diff
@@ -12,14 +12,16 @@ import {
   MetroSerializer as MetroSerializerEsbuild,
 } from "@rnx-kit/metro-serializer-esbuild";
 import type { AllPlatforms } from "@rnx-kit/tools-react-native/platform";
+import { changeHostToUseReactNativeResolver } from "@rnx-kit/typescript-react-native-resolver";
 import {
   Project,
   readConfigFile,
   createDiagnosticWriter,
 } from "@rnx-kit/typescript-service";
 import type { DeltaResult, Graph } from "metro";
 import type { InputConfigT, SerializerConfigT } from "metro-config";
-import { MetroTypeScriptResolverHost } from "./metro-ts-resolver";
+import ts from "typescript";
```

Comment:
Can we import the type only?
```suggestion
import type { LanguageServiceHost } from "typescript";
```